packages/block-library/src/buttons/edit.js

```diff
@@ -10,14 +10,13 @@ import {
 	BlockControls,
 	useBlockProps,
 	__experimentalUseInnerBlocksProps as useInnerBlocksProps,
+	JustifyToolbar,
 } from '@wordpress/block-editor';
-import { ToolbarGroup, ToolbarItem } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { name as buttonBlockName } from '../button';
-import ContentJustificationDropdown from './content-justification-dropdown';
```

Comment:
`ContentJustificationDropdown` component was used only in `buttons`, so we should remove this.

Also the above component is used in `native` file, so it should be replaced there as well and ping someone who can test this.

Reply:
It looks like it is stilled used in edit.native.js in buttons. I think I can make the same update there and then remove, but will need to read up on how to test for mobile. I can do it under a different PR and have some one from mobile assist.